tfhe/src/shortint/glwe_compression/seeded_server_keys.rs

```diff
@@ -0,0 +1,122 @@
+use super::{GlweCompressionKey, GlweCompressionPrivateKeys, GlweDecompressionKey};
+use crate::core_crypto::prelude::{
+    allocate_and_generate_new_seeded_lwe_packing_keyswitch_key,
+    par_allocate_and_generate_new_seeded_lwe_bootstrap_key,
+    par_convert_standard_lwe_bootstrap_key_to_fourier, FourierLweBootstrapKey,
+    SeededLweBootstrapKeyOwned, SeededLwePackingKeyswitchKey,
+};
+use crate::shortint::client_key::ClientKey;
+use crate::shortint::engine::ShortintEngine;
+use crate::shortint::parameters::PolynomialSize;
+use crate::shortint::server_key::ShortintBootstrappingKey;
+use crate::shortint::{ClassicPBSParameters, EncryptionKeyChoice, PBSParameters};
+use serde::{Deserialize, Serialize};
+use std::fmt::Debug;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct SeededGlweCompressionKey {
```

Comment:
Seeded is not the wording used in shortint and higher, CompressedGlweCompressionKey

Reply:
Renamed